ci/github/helpers/build_docker_image_tag

```diff
@@ -0,0 +1,34 @@
+#!/bin/bash
+# Usage: build_docker_image_tag
+# returns the slugified name of the image tag that shall be used
+# e.g.: current git branch name
+# if on travis,
+#   if on a branch: returns the name of the travis branch
+#   if on a pull request: returns the name of the originating branch
+#   if on master: returns "build"
+
+# http://redsymbol.net/articles/unofficial-bash-strict-mode/
+set -euo pipefail
+IFS=$'\n\t'
+
+if [ ! -v GITHUB_ACTIONS ] || [ $GITHUB_ACTIONS = "false" ]; then
+    # no travis here so let's use the git name directly
+    image_tag=$(git rev-parse --abbrev-ref HEAD)
+else
+    branch_name=${GITHUB_REF##*/}
+    if [ "$branch_name" = "master" ]; then
```

Comment:
As the same code is executed in both conditions, maybe using a -o would be more convenient ?

Reply:
that's right. since the github action does not yet deploy I left it open.